pkgs/desktops/cinnamon/xapps/default.nix

```diff
@@ -0,0 +1,79 @@
+{ fetchFromGitHub
+, fetchpatch
+, glib
+, gobject-introspection
+, gtk3
+, libgnomekbd
+, libxklavier
+, meson
+, ninja
+, pkgconfig
+, python3
+, python3Packages
+, stdenv
+, vala
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xapps";
+  version = "1.6.4";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = pname;
+    rev = version;
+    sha256 = "08c8j7wv56zf8mfgjfhd0wc1bbrw0dj1lmlawvsxwfajnnyjsm0d";
+  };
+
+  NIX_CFLAGS_COMPILE = [ "-I${glib.dev}/include/gio-unix-2.0" ]; # TODO: https://github.com/NixOS/nixpkgs/issues/36468
+
+  patches = [
+    (fetchpatch { # details see https://github.com/linuxmint/xapps/pull/65
+      url = "https://github.com/linuxmint/xapps/compare/d361d9cf357fade59b4bb68df2dcb2c0c39f90e1...2dfe82ec68981ea046345b2be349bd56293579f7.diff";
+      sha256 = "0sffclamvjas8ad57kxrg0vrgrd95xsk0xdl53dc3yivpxkfxrnk";
+    })
+  ];
+
+  buildInputs = [
+    glib
+    gtk3
+    libgnomekbd
+    libxklavier
+    (python3.withPackages(ps: with ps; [ pygobject3 ]))
+    python3Packages.pygobject3 # otherwise pkgconfig can't find it
+  ];
+
+  nativeBuildInputs = [
+    meson
+    gobject-introspection
+    ninja
+    wrapGAppsHook
+    pkgconfig
+    vala
+  ];
+
+  mesonFlags = [
+    "-Dpy-overrides-dir=${placeholder "out"}/${python3.sitePackages}/gi/overrides"
+  ];
+
+  postPatch = ''
+    chmod +x schemas/meson_install_schemas.py # patchShebangs requires executable file
+
+    # seems to be broken in PR, please fix @worldofpeace
+    mv ./files/usr/bin/pastebin ./scripts/pastebin
+    mv ./files/usr/bin/upload-system-info ./scripts/upload-system-info
+    mv ./files/usr/bin/xfce4-set-wallpaper ./scripts/xfce4-set-wallpaper
```

Comment:
When I check my github clone these files are in the correct locations. I think it's because how `patchPhase` applies patches. I guess this is fine for now, but could you note it's "The patch application doesn't result in these files being moved properly"?

Reply:
Added note